test/mailers/notifier_mailer_test.rb

```diff
@@ -138,14 +138,27 @@ class NotifierMailerTest < ActionMailer::TestCase
     assert response.subject.include?(
       I18n.t(
         'notifier.notify_new_finding_answer.title',
-        :review => finding_answer.finding.review.to_s
+        review: finding_answer.finding.review,
+        finding_id: finding_answer.finding.id
       )
     )
     assert_match Regexp.new(I18n.t('notifier.notify_new_finding_answer.finding_link')),
       response.body.decoded
     assert_equal user.email, response.to.first
   end
 
+  test 'notify action not found' do
+    user    = User.find(users(:administrator).id)
```

Comment:
Esto estaba mal de antes, debería ir así (funciona de las dos maneras, pero acá hace dos búsquedas).
```suggestion
    user    = users(:administrator)
```

Reply:
dale genial, se lo cambio al de arriba tmb